mass_mailing_custom_unsubscribe/views/mail_unsubscription_view.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
+
+<record id="mail_unsubscription_view_form" model="ir.ui.view">
+    <field name="name">Mail Unsubscription Form</field>
+    <field name="model">mail.unsubscription</field>
+    <field name="arch" type="xml">
+        <form>
+            <sheet>
+                <group>
+                    <field name="date"/>
+                    <field name="mass_mailing_id"/>
+                    <field name="unsubscriber_id"/>
+                    <field name="email"/>
+                    <field name="success"/>
+                    <field name="reason_id"/>
+                    <field
```

Comment:
Although there's no guideline about this, the way you indent XML files makes Odoo to treat this very weird. Some points:

* All that goes inside an attribute value should be in the same line, or it will be parsed it a lot of spaces (and not respecting the logical indentation you see in the file), so for example, please put `attrs="{'required': [('details_required', '=', True)]}"` 
* Most editors put the first attribute in the same line of the tag , and the second aligned with the first letter of this attribute. I like this convention because it saves one line as it's still very readable. For example:
```
<field name="details"
       attrs="{'req...
```